tools/database-converter/index.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * This script can be used to convert databases from one format to another
+ * Format:
+ * `node tools/database-converter --database <database> --from <database format> --to <database format>`
+ * Usage example:
+ * `node tools/database-converter --database punishments --from tsv --to sqlite`
+ */
+
+'use strict';
+
+const child_process = require('child_process');
+child_process.execSync('node build');
+
+const PunishmentsConverter = require('./punishments-converter').PunishmentsConverter;
+
+/** @type {{[k: string]: string[]}} */
+const DATABASES = {
+	punishments: ['tsv', 'sqlite'],
+};
+
+function parseFlags() {
```

Comment:
Given the increased need for options processing I think we should just add a devDependency on minimist or similar (minimist just happens to have a [great dependency graph](http://npm.broofa.com/?q=minimist)...)